### DISCUSSION TITLE
Company Portal app not working with mitmproxy

### DISCUSSION THREAD

**Discussion by changsin** [mitmproxy/mitmproxy#6825] [General]
I am seeing quite weird when trying to use Microsoft intune's Company Portal app.
The app runs fine without the proxy. But it cannot connect to the server once it's turned on.

Repro steps:
1. Install Company Portal app from Microsoft Store
2. Run mitmproxy

Result:
The Company Portal app cannot sign in the user.
It tries to connect to login.microsoftonline.com
But I cannot see this URL from the mitmproxy output (no attempt to connect)

I tried to put login.microsoftonline.com in the ignoreHosts list, but it had no effect.

Mitmproxy: 10.3.0
Python: 3.10.11
OpenSSL: OpenSSL 3.2.1 30 Jan 2024
Platform: Windows-10-10.0.22631-SP0

**Comment by canda** [comment]
In what mode are you running mitmproxy?

**Comment by emanuele-em** [comment]
From what I unerstood, it sounds like you didn't install the certificates, once you set up the proxy, remember to go to mitm.it and download the certificate for your device, then install it

**Comment by changsin** [comment]
Actually, I installed it (the certificate), and it works fine for other websites. I even ignored everything by specifying .*, but it still failed. Once I placed login.microsoftonline.com and *.microsoft.com to the Windows ProxyOverride list, it worked. So it is something that mitmproxy is doing.
